reference/reference.md

```diff
@@ -10,3 +10,5 @@ developers who want to extend features in Valtimo and need to know what is avail
 * [User interface](./user-interface/user-interface.md)
 * [Feature toggles](feature-toggles.md)
 * [Troubleshoot](troubleshoot.md)
+* [Getting started back-end](https://github.com/valtimo-platform/valtimo-backend-libraries#readme)
+* [Getting started front-end](https://github.com/valtimo-platform/valtimo-frontend-libraries#readme)
```

Comment:
I feel like this misses a bit of context. Maybe add these links to a new page called 'Open source repositories'. 
And link that new page here

Reply:
Hmm  I just saw your story is about 'starting up Valtimo locally'. I'm wondering where the correct location is for that kind of documentation.

Also the 'Getting started ...' is a bit confusing since we use that for something else already. [here](https://docs.valtimo.nl/getting-started/first-dive/creating-your-own-valtimo-implementation)

Reply:
Yeah I'm a bit confused as well because this is not for creating an implementation but running the libraries locally, so you can contribute as an open source developer.

Reply:
@ThomasRitense Do you have an idea on where to put this?

Reply:
It should be in the getting started -> first dive in a new section. Something like 'run the libraries'?